browser/model/messages.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
I would say this information belongs to requirements.json file. It would be good to be able to update all info by downloading only one file from remote location (part of https://issues.jboss.org/browse/JBDS-4470).